utility/src/main/java/org/oppia/android/util/math/FractionParser.kt

```diff
@@ -24,6 +25,9 @@ class FractionParser {
    * detection should be done using [getRealTimeAnswerError], instead.
    */
   fun getSubmitTimeError(text: String): FractionParsingError {
+    if (TextUtils.isEmpty(text)) {
```

Comment:
Not really. TextUtils also checks for nullness, so in the android-Java world it is safer to use, and I keep using it without thinking too much. I don't mind switching to kotlin's isEmpty if you think it is better.

Reply:
In the project, we mainly use kotlin extension functions for things like these, even though the functionality is basically the same. It is up to you to change it.

If you want to consider a string with only whitespace characters as empty, use isBlank().

Reply:
Kotlin also has ``isNullOrBlank`` but the nullness check isn't necessary since we can statically guarantee non-nullness at this point.

I think I also lean toward not using TextUtils here. The more we depend on Android in the utility module the harder it will be to abstract away Android functionality in the future (so we should only depend on Android when absolutely necessary) in the event we want to support more than just Android with this codebase.

Reply:
Changed to isNullOrBlank.